www/src/providers/Auth0ProviderWithHistory.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
What is the point of this component. If I understand correctly it just makes sure that we can go back to the original page after logging in?

Reply:
I was imagining the case when the refresh token expires and an explicit re-sign in is required if the user has left the site on for a while. But then in that case we could also just redirect the user back to a fixed page (i.e. the dashboard)

Reply:
:+1: